src/DynamoCoreWpf/ViewModels/Preview/WatchViewModel.cs

```diff
@@ -313,6 +321,12 @@ private string GetDisplayType(object obj)
                     return nameof(TypeCode.Object);
                 case TypeCode.String:
                     return nameof(TypeCode.String);
+                case TypeCode.Byte:
+                    return nameof(TypeCode.Byte);
+                case TypeCode.UInt32:
+                    return nameof(TypeCode.UInt32);
+                case TypeCode.UInt64:
+                    return nameof(TypeCode.UInt64);
```

Comment:
I don't believe the type returned from `GetDisplayType` is actually being used currently so it might just be better to remove this for function for the time being.